google/resource_container_cluster.go

```diff
@@ -610,6 +610,21 @@ func resourceContainerCluster() *schema.Resource {
 				},
 			},
 
+			"vertical_pod_autoscaling": {
+				Type:     schema.TypeList,
+				MaxItems: 1,
+				Optional: true,
+				Computed: true,
```

Comment:
We try to avoid setting something as `Computed` unless there's a default that is sent back from the API when nothing is passed. Is vertical_pod_autoscaling part of the API response even if this isn't present and it hasn't been configured upstream?

Reply:
Thanks, that was just an artifact of me cribbing off of other definitions. I've removed it.